bin/Sherpa/cards/production/13TeV/DiphotonNLO/Run.dat_yyNLO

```diff
@@ -0,0 +1,60 @@
+(run){
+  EVENTS 1000;
+  EVENT_MODE HepMC;
+
+  % scales, tags for scale variations
+  FSF:=1.; RSF:=1.; QSF:=1.;
+  SCALES STRICT_METS{FSF*MU_F2}{RSF*MU_R2}{QSF*MU_Q2};
+  HEPMC_USE_NAMED_WEIGHTS=1;
+  SCALE_VARIATIONS=0.25,0.25 0.25,1. 1.,0.25 1.,1. 1.,4. 4.,1. 4.,4.;
+  PDF_VARIATIONS=NNPDF31_nnlo_as_0118_mc_hessian_pdfas[all] NNPDF30_nnlo_as_0118_hessian
+  ALPHAQED_DEFAULT_SCALE=0.0
+
+  % tags for process setup
+  NJET:=3; LJET:=2,3; QCUT:=10;
+
+  % Beam and  PDF settings
+  BEAM_1 2212; BEAM_ENERGY_1 6500.;
+  BEAM_2 2212; BEAM_ENERGY_2 6500.;
+
+  PDF_LIBRARY     LHAPDFSherpa;
+  PDF_SET         NNPDF31_nnlo_as_0118_mc_hessian_pdfas;
+  PDF_SET_VERSION 0;
+  PDF_GRID_PATH   PDFsets;
+
+  % me generator settings
+  ME_SIGNAL_GENERATOR Amegic Comix LOOPGEN Internal;
+  LOOPGEN:=OpenLoops;
+  AMEGIC_LIBRARY_MODE=0
+  OL_PARAMETERS=redlib1=5=redlib2=5=write_parameters=1
+}(run)
+
+(processes){
+  Process 21 21 -> 22 22
+  ME_Generator Internal;
+  Loop_Generator gg_yy
+  Scales VAR{FSF*Abs2(p[2]+p[3])}{RSF*Abs2(p[2]+p[3])}{QSF*Abs2(p[2]+p[3])};
+  End process;
+
+  Process 93 93 -> 22 22 93{NJET};
+  Order (*,2);
+  CKKW sqr(QCUT/E_CMS)/(1.0+sqr(QCUT/0.6)/Abs2(p[2]+p[3]));
+  NLO_QCD_Mode MC@NLO {LJET};
+  ME_Generator Amegic {LJET};
+  RS_ME_Generator Comix {LJET};
+  Loop_Generator LOOPGEN {LJET};
+  PSI_ItMin 50000 {3}
+  Integration_Error 0.99 {3}
```

Comment:
0.99 should be a typo?

Reply:
it's actually intended - we are following the configuration used by ATLAS, and we understand this is what is recommended by their sherpa experts. Our understanding is the `PSI_ItMin` setting is effectively controlling the accuracy here, and indeed we see precisions on the {3,4,5} subprocesses of < 2% from the sherpa logs.